app/src/sdk/BffApiClient.js

```diff
@@ -28,6 +28,17 @@ export class BffApiClient {
     return { success: true, ...body };
   }
 
+  async auth0Authenticate(token) {
+    const result = await this.post("/api/oauth/login", { token });
+
+    if (result.status !== 200) {
+      return { success: false, token: null, userDetails: null };
+    }
+
+    const body = await result.json();
```

Comment:
Should we consider using axios or other alternatives to fetch to avoid having to perform `.json()` every time?